src/components/Categories/Categories.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { useProductCategories } from "../../utils/hooks/useProductCategories";
+import Card from "../Card/Card";
+
+const ProductCategories = () => {
+  const { data, isLoading } = useProductCategories();
+  if (isLoading) {
+    return <div>{"..."}</div>;
+  }
+
+  return (
+    <section>
+      <h4>Categories</h4>
+      <div>
+        {data.results.map((category) => (
+          <Card
+            key={category.id}
+            id={category.id}
+            name={category.data.name}
+            image={category.data.main_image.url}
+          />
+        ))}
```

Comment:
algo que en lo personal algunas veces es mejor es destructurar las propiedades para evitar repetir mucho el `category.` por ejemplo
```suggestion
        {data.results.map(({id, data}) => (
          <Card
            key={id}
            id={id}
            name={data.name}
            image={data.main_image.url}
          />
        ))}
```